modules/godot_physics_3d/register_types.cpp

```diff
@@ -51,7 +53,11 @@ void initialize_godot_physics_3d_module(ModuleInitializationLevel p_level) {
 		return;
 	}
 	PhysicsServer3DManager::get_singleton()->register_server("GodotPhysics3D", callable_mp_static(_createGodotPhysics3DCallback));
+#ifndef MODULE_JOLT_PHYSICS_ENABLED
+	// If Jolt Physics are disabled at compile-time, set GodotPhysics3D as the default server.
```

Comment:
It's not particularly "intrusive" but is this extra line necessary? The code speaks for itself.